wagtailquickcreate/tests/tests.py

```diff
@@ -33,14 +35,33 @@ def test_admin(self):
 
     def test_quickcreate_panel_links(self):
         response = self.client.get('/admin/')
-        self.assertTrue('Add Image' in str(response.content))
-        self.assertTrue('Add Document' in str(response.content))
-        self.assertTrue('Add Information page' in str(response.content))
+        soup = BeautifulSoup(response.content, 'html.parser')
+        quick_create_content = soup.find('div', id='quickcreate-content')
```

Comment:
I'm going to leave this as it is. 
The id is required, as far as I can tell for the toggle button to work, if the id was changed then without a test, that would fail, it could be missed. e.g. The toggle button stops working and you'd need to actually use the toggle button to notice that.
I could add further tests but I think the package is relatively simple and doesn't really need them.

Reply:
Sounds good, I am just fulfilling my obligation to James Hancock of spreading the `data-testid` gospel 😆